app/templates/auth/login.html

```diff
@@ -47,7 +47,7 @@ <h3 class="validation-masthead-heading" id="validation-masthead-heading">
     <h1>Supplier login</h1>
 </header>
 
-<form action="{{ url_for('.process_login', next=next) }}" method="POST">
+<form autocomplete="off" action="{{ url_for('.process_login', next=next) }}" method="POST">
```

Comment:
Does it need to be on both the form and the individual fields?  Seems like either one should do on its own?